src/components/PlaidLink/index.native.js

```diff
@@ -1,11 +1,10 @@
 import {useEffect} from 'react';
-import {dismissLink, openLink, useDeepLinkRedirector, usePlaidEmitter} from 'react-native-plaid-link-sdk';
+import {dismissLink, openLink, usePlaidEmitter} from 'react-native-plaid-link-sdk';
 import Log from '@libs/Log';
 import CONST from '@src/CONST';
 import {plaidLinkDefaultProps, plaidLinkPropTypes} from './plaidLinkPropTypes';
 
 function PlaidLink(props) {
-    useDeepLinkRedirector();
```

Comment:
`useDeepLinkRedirector` was removed from `react-native-plaid-link-sdk` since it's no longer needed.

Reply:
Yes. There is not such hook in new version and it is not need. It is mentioned in plaid link official doc